core/tests/test_images.py

```diff
@@ -120,3 +120,23 @@ def test_image_save():
 
     image.save(save_path)
     assert_action_performed_with(image, "save", path=save_path)
+
+def test_pil_support():
+    from PIL import Image as PIL_Image
+
+    pil_img = PIL_Image.open("resources/toga.png")
+    toga_img = toga.Image("resources/toga.png")
+    toga_img_from_pil_img = toga.Image(pil_image = pil_img)
+
+    assert toga_img.width == toga_img_from_pil_img.width, "PIL support is faulty"
```

Comment:
We know what the reference image is, so we don't need to compare with the same image loaded a different way - we can compare the size directly with the known values.